oneko.js

```diff
@@ -234,12 +238,17 @@
         nekoEl.style.top = `${nekoPosY - 16}px`;
       };
 
-      const mouseup = () => {
+      const mouseup = (e) => {
         grabbing = false;
         nudge = true;
         resetIdleAnimation();
         window.removeEventListener("mousemove", mousemove);
         window.removeEventListener("mouseup", mouseup);
+        // Refresh sleeping position when dragged away
+        mousePosX = e.clientX;
+        sleepSaved = false;
+        forceSleep = false;
+        sleep();
```

Comment:
This would force her to sleep after dragging even when she's not put to sleep before.